### PR TITLE
Introduce slim track validation sequences 

### PR DESCRIPTION
**Pull request by lveldere**
These sequences only depend on the output of iterative tracking and on tracking particles.

For e.g. FastSim these sequences can be run by calling cmsDriver.py with the following steps options:
-s RECOBEFMIX,DIGI:pdigi_valid,VALIDATION:tracksValidationSlim
-s RECOBEFMIX,DIGI:pdigi_valid,VALIDATION:tracksAndTrajectorySeedsValidationSlim

@makortel 
Please let me know if you think this is a bad idea, or needs changes.
For FastSim this is definitely very useful.

### CODE REVIEW

**Comment by lveldere** [comment]
please test

File: Validation/RecoTrack/python/TrackValidation_cff.py [hunk 1]
```diff
@@ -340,3 +340,23 @@
     trackValidatorAllTPEfficStandalone
 )
 
+# 'slim' sequences that only depend on track and tracking particle collections
+tracksValidationSelectorsSlim = tracksValidationSelectors.copyAndExclude([cutsRecoTracksBtvLike,ak4JetTracksAssociatorAtVertexPFAll,cutsRecoTracksAK4PFJets])
+
+tracksPreValidationSlim = cms.Sequence(
+    tracksValidationSelectorsSlim +
+    tracksValidationTruth +
+    tracksValidationTruthSignal
```
Comment (makortel) [inline Validation/RecoTrack/python/TrackValidation_cff.py]: It looks to me that you don't need `tracksValidationTruthSignal` sequence here.

**Comment by makortel** [comment]
@lveldere I'm fine with the changes, but I'm not sure of the word "slim". I don't have better suggestions though.

**Comment by lveldere** [comment]
I can't think of anything that covers things better than 'Slim'.
'Standalone' is taken and I'm not sure if 'Pure' or 'Minimal' are any
better than 'Slim'.
As soon if you can give me a green light on the naming,
I'll get rid of the tracksValidationTruthSignal

**Comment by makortel** [comment]
Let's proceed with it, and change it later if needed.

**Comment by lveldere** [comment]
please test

thanks @makortel

**Comment by deguio** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**